zeebe/engine/src/main/java/io/camunda/zeebe/engine/state/authorization/PersistedPermissions.java

```diff
@@ -17,12 +17,12 @@
 import java.util.stream.StreamSupport;
 
 public class PersistedPermissions extends UnpackedObject implements DbValue {
-  private final ArrayProperty<StringValue> permissionsProp =
-      new ArrayProperty<>("permissions", StringValue::new);
+  private final ArrayProperty<StringValue> resourceIds =
```

Comment:
❓ Same question regarding the `id` vs `key` conversation

Reply:
Same answer 😄